app/gunicorn_config.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+import traceback
+
+workers = 10
+errorlog = "/home/vcap/logs/gunicorn_error.log"
```

Comment:
do we want to move the dockerfile working directory so that the app is in `/home/vcap/app`?

Reply:
I don't see why not 👍